### ISSUE TITLE
.deb binary package has invalid version field

### ISSUE THREAD

**Issue by JonnieCache** [honeycombio/honeymarker#64]
**Versions**

- Go: N/A
- Honeymarker: 0.2.9
- dpkg: 1.19.7

**Steps to reproduce**

1. `wget "https://github.com/honeycombio/honeymarker/releases/download/v0.2.9/honeymarker_v0.2.9_amd64.deb"`
2. `sudo dpkg -i honeymarker_v0.2.9_amd64.deb`

**Additional context**

Gives the following output:

```
dpkg: error processing archive honeymarker_v0.2.9_amd64.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 2 package 'honeymarker':
 'Version' field value 'v0.2.9': version number does not start with digit
Errors were encountered while processing:
 honeymarker_v0.2.9_amd64.deb
```

Apparently this policy is enforced after dpkg version 1.16. I'm on 1.19.7 on ubuntu 20.04 (focal LTS.)

Apparently the `v` needs to be stripped.

**Comment by JonnieCache** [comment]
Ah I see this has been remedied already...

**Comment by JonnieCache** [comment]
Hmm it appears that the fix should have been applied for 0.2.9 already? So I'm reopening this.

**Comment by pbiggar** [comment]
I'm getting the same error except for v0.2.10
